core-5/src/main/java/configuration/car/carImpl/ChevroletCamaro.java

```diff
@@ -0,0 +1,30 @@
+package configuration.car.carImpl;
+
+import configuration.car.Car;
+import configuration.engine.Engine;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+
+public class ChevroletCamaro implements Car {
+    private Engine engine;
+
+    private int maxSpeed;
+
+    @Autowired
+    public void setMaxSpeed(@Value("${chevrolet.maxSpeed:300}")int maxSpeed){
```

Comment:
if you go with configuration class, lets wire it at the same place, not inside the class